m4_sessions/1711-inning/Group_5.md

```diff
@@ -1,9 +1,30 @@
 Group Five
 
-Session Title: Description: Why should students come to this session?
+Session Title: Technology in the Media
+
+Description: Why should students come to this session?
+
+With terms like 'Russian hacking', 'bitcoin', 'net nuetrality' and 'fake news' dominating the news cycle, what can we as developers do to help aid in understanding what these buzzwords actually mean. 
 
 Session Outcomes: What will students leave with or gain from this session?
 
+Students will be able to respond to questions concerning technology topics in the media. Students will lead what we can do as developers to help bridge the understanding for people who lack technical literacy. 
+
 Session Outline: Include the flow of the session and the time for each section.
 
-Names of group members:
+Class intro: 5 minutes
+Have students introduce themselves, brainstorm technological topics in the news that they do not know much about. 
+
+Pairs: 10 minutes
+Get in pairs and discuss 
+
+Gear up leaders share: 10 minutes
+Explain how we feel about some of these buzzword issues, and share our thoughts on how to explain the concepts to a 5 year old. 
+
+This is as far as we got. Will expand after we do research indepedently. 
```

Comment:
Totally understand....think about including some small group discussion, sharing resources, and closing the session in a meaningful way.